test_communication/package.xml

```diff
@@ -20,7 +20,7 @@
   <exec_depend>rosidl_default_runtime</exec_depend>
 
   <test_depend>ament_cmake</test_depend>
-  <test_depend>ament_cmake_nose</test_depend>
+  <test_depend>ament_cmake_pytest</test_depend>
```

Comment:
Nit: alphabetical order

Reply:
I didn't notice that they were out-of-order before already. Anyway fixed.